package.json

```diff
@@ -24,6 +24,7 @@
     "commander": "2.13.0",
     "elasticsearch": "14.1.0",
     "express": "4.16.2",
+    "json-patch": "^0.7.0",
```

Comment:
Probably want to version lock this, for consistency with the other dependencies.